common/json.c

```diff
@@ -122,22 +147,29 @@ bool json_to_u32(const char *buffer, const jsmntok_t *tok,
 
 bool json_to_int(const char *buffer, const jsmntok_t *tok, int *num)
 {
-	char *end;
-	long l;
+	int64_t s64;
```

Comment:
s/`int64_t`/`s64`/ as well.  Also the variable name probably needs to be changed because `s64 s64` is bleah.

Reply:
Done in fb8ab0aa.